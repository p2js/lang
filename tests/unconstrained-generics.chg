function foo<T>(): i32 {
    return 42;
}

function main(): i32 {
    return foo();
}
