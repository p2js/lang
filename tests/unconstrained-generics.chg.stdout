expr @ 2:12 2:14 => i32
expr @ 2:5 2:14 => never
expr @ 1:24 4:0 => void
expr @ 6:12 6:15 => fn foo(...)
expr @ 6:12 6:17 => i32
expr @ 6:5 6:17 => never
expr @ 5:22 8:1 => void

